packages/grafana-ui/src/components/TimePicker/TimeRangePicker/TimePickerContent.tsx

```diff
@@ -11,58 +11,60 @@ import { TimeRangeForm } from './TimeRangeForm';
 import { TimeRangeList } from './TimeRangeList';
 import { TimePickerFooter } from './TimePickerFooter';
 
-const getStyles = stylesFactory((theme: GrafanaTheme, isReversed) => {
-  const containerBorder = theme.isDark ? theme.palette.dark9 : theme.palette.gray5;
+const getStyles = stylesFactory(
+  (theme: GrafanaTheme, isFullscreen, isReversed, hideQuickRanges, showHistory, isHistoryEmpty) => {
```

Comment:
This seems like too many params for a style function and we're using most of them only once on line 36, so maybe they could be combined into one param outside of this function?

Reply:
I thought about this and I was wondering if the computations should be performed in the component or the `getStyles` method. My rationale was to avoid increasing complexity in the component itself.